DEPS.bzl

```diff
@@ -2429,8 +2429,9 @@ def go_deps():
         name = "com_github_lib_pq",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/lib/pq",
-        sum = "h1:9xohqzkUwzR4Ga4ivdTcawVS89YSDVxXMa3xJX3cGzg=",
-        version = "v1.8.0",
+        replace = "github.com/cockroachdb/pq",
```

Comment:
https://github.com/lib/pq/pull/1039 was merged so should we just tag/release that?

Reply:
see the comment in go.mod below -- we can't use lib/pq on master because of #65320 ...